gnucash/gnome/gnc-plugin-page-register.c

```diff
@@ -3405,22 +3405,22 @@ gnc_plugin_page_register_set_filter_tooltip (GncPluginPageRegister* page)
     {
         gchar* sdate = qof_print_date (priv->fd.start_time);
         t_list = g_list_prepend
-            (t_list, g_strdup_printf ("%s %s", _("Start Date:"), sdate));
+            (t_list, g_strdup_printf ("%s: %s", _("Start Date"), sdate));
```

Comment:
No, different languages use punctuation differently, and separating the msgid from the format is bad because it stops translators from reordering it. It should be `_("Start Date: %s")`.

Reply:
> No, different languages use punctuation differently, and separating the msgid from the format is bad because it stops translators from reordering it.

I tried to get less messages, but you are right, in fr there should be a nbsp between term and colon.

IIRC there are more hardcoded colons, mostly in reports. Perhaps a separate const string for the colon would be a general solution.?
> It should be `_("Start Date: %s")`.

Reply:
> IIRC there are more hardcoded colons, mostly in reports. Perhaps a separate const string for the colon would be a general solution.?

[Not really](https://www.gnu.org/software/gettext/manual/gettext.html#Preparing-Strings), and that doesn't even mention that `printf` formatting is deficient because the translator can't reorder the inserted tokens, never mind that they're not descriptive and we often don't even provide comments explaining what gets put where.